sdk/go/common/workspace/plugins.go

```diff
@@ -2129,28 +2162,6 @@ func (sp SortedPluginInfo) Less(i, j int) bool {
 }
 func (sp SortedPluginInfo) Swap(i, j int) { sp[i], sp[j] = sp[j], sp[i] }
 
-// SortedPluginSpec is a wrapper around PluginSpec that allows for sorting by version.
```

Comment:
This is no longer needed, so I've deleted it. While it's technically public in `sdk/go/common`, I'm pretty sure this is safe -- https://github.com/search?q=%22SortedPluginSpec%22+language%3AGo&type=code returns no results across the whole of GitHub that aren't forks of Pulumi and our single use of it prior to this commit.